src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,43 +1,135 @@
+import { memo, useMemo, useState } from 'react'
 import { useTranslate } from 'react-polyglot'
 import { Route, Switch, useRouteMatch } from 'react-router'
+import SwipeableViews from 'react-swipeable-views'
+import { mod } from 'react-swipeable-views-core'
+import { type SlideRenderProps, virtualize } from 'react-swipeable-views-utils'
 import { Main } from 'components/Layout/Main'
 import { SEO } from 'components/Layout/Seo'
 import { NftTable } from 'components/Nfts/NftTable'
-import { useFetchOpportunities } from 'components/StakingVaults/hooks/useFetchOpporunities'
 import { RawText } from 'components/Text'
 import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+import { isMobile } from 'lib/globals'
 import { Accounts } from 'pages/Accounts/Accounts'
 import { TransactionHistory } from 'pages/TransactionHistory/TransactionHistory'
+import { breakpoints } from 'theme/theme'
 
-import { DashboardHeader } from './components/DashboardHeader'
+import { DashboardHeader } from './components/DashboardHeader/DashboardHeader'
 import { DashboardSidebar } from './DashboardSidebar'
 import { EarnDashboard } from './EarnDashboard'
+import { MobileActivity } from './MobileActivity'
 import { Portfolio } from './Portfolio'
 import { RewardsDashboard } from './RewardsDashboard'
 import { WalletDashboard } from './WalletDashboard'
 
 const direction: StackDirection = { base: 'column', xl: 'row' }
 const maxWidth = { base: 'full', lg: 'full', xl: 'sm' }
+const mainPadding = { base: 0, md: 4 }
+const customTabActive = { color: 'text.base' }
+const customTabLast = { marginRight: 0 }
+const pageProps = { paddingTop: 0 }
+const CustomTab = (props: TabProps) => (
+  <Tab
+    fontWeight='semibold'
+    color='text.subtle'
+    _selected={customTabActive}
+    px={0}
+    py={4}
+    mr={6}
+    _last={customTabLast}
+    {...props}
+  />
+)
+
+const ScrollView = (props: FlexProps) => (
+  <Flex
+    flexDir='column'
+    width='100vw'
+    pt='calc(var(--mobile-header-offset) + 1rem)'
+    pb='var(--mobile-nav-offset)'
+    height='100vh'
+    overflowY='auto'
+    {...props}
+  />
+)
+
+const VirtualizedSwipableViews = virtualize(SwipeableViews)
 
 export const Dashboard = memo(() => {
   const translate = useTranslate()
+  const [slideIndex, setSlideIndex] = useState(0)
+  const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`, { ssr: false })
   const isDefiDashboardEnabled = useFeatureFlag('DefiDashboard')
   const { path } = useRouteMatch()
   const isNftsEnabled = useFeatureFlag('Jaypegz')
+  const appIsMobile = isMobile || !isLargerThanMd
+
+  const handleSlideChange = (index: number) => {
+    setSlideIndex(index)
+  }
 
-  const { isLoading: isOpportunitiesLoading } = useFetchOpportunities()
+  const mobileTabs = useMemo(() => {
+    return (
+      <Tabs mx={6} index={slideIndex} variant='unstyled' onChange={handleSlideChange}>
```

Comment:
If you want to save 2 lines
```suggestion
      <Tabs mx={6} index={slideIndex} variant='unstyled' onChange={setSlideIndex}>
```